monkey/tests/unit_tests/monkey_island/cc/services/ransomware/test_ransomware_report.py

```diff
@@ -0,0 +1,70 @@
+import mongoengine
+import pytest
+from mongoengine import get_connection
+from tests.data_for_tests.mongo_documents.edges import EDGE_EXPLOITED, EDGE_SCANNED
+from tests.data_for_tests.mongo_documents.monkeys import MONKEY_AT_ISLAND, MONKEY_AT_VICTIM
+from tests.data_for_tests.mongo_documents.telemetries.file_encryption import (
+    ENCRYPTED,
+    ENCRYPTED_2,
+    ENCRYPTION_ERROR,
+    ENCRYPTION_ONE_FILE,
+)
+
+from monkey_island.cc.services.ransomware.ransomware_report import RansomwareReportService
+
+
+@pytest.fixture
+def fake_mongo(monkeypatch):
+    mongoengine.connect("mongoenginetest", host="mongomock://localhost")
+    mongo = get_connection()
+    monkeypatch.setattr("monkey_island.cc.services.ransomware.ransomware_report.mongo", mongo)
+    monkeypatch.setattr("monkey_island.cc.services.reporting.report.mongo", mongo)
+    monkeypatch.setattr("monkey_island.cc.services.node.mongo", mongo)
+    return mongo
```

Comment:
I need help with this one. 
`RansomwareReportService` uses `ReportService` which further down the road uses `Monkey` model. I can't seem to find a way to force the model to use the same database as we use here.